### PR TITLE
Update ember-cli to 2.5.0

### PR DESCRIPTION
**Pull request by mike-north**

### CODE REVIEW

**Comment by mike-north** [comment]
this appears to be fine with fastboot 1.0.0-beta.1 now

**Comment by ivanvanderbyl** [comment]
:shipit:

**Comment by mike-north** [comment]
@ivanvanderbyl now mergable

File: tests/dummy/app/components/d3-donut/component.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
+import { arc, pie } from  'd3-shape';
```
Comment (ivanvanderbyl) [inline tests/dummy/app/components/d3-donut/component.js]: Remove this file. I removed it from `master`.

Comment (mike-north) [inline tests/dummy/app/components/d3-donut/component.js]: done

**Comment by ivanvanderbyl** [comment]
Thanks @mike-north  :D

**Comment by mike-north** [comment]
would love a new release :)

**Comment by ivanvanderbyl** [comment]
Cutting one now.

**Comment by ivanvanderbyl** [comment]
Released https://github.com/ivanvanderbyl/ember-cli-d3-shape/releases/tag/v0.8.5

**Pull request merged**